src/main/java/uk/gov/hmcts/reform/prl/constants/ManageDocumentsCategoryConstants.java

```diff
@@ -56,6 +56,7 @@ public class ManageDocumentsCategoryConstants {
     public static final String SPECIAL_MEASURES = "specialMeasures";
     public static final String ANY_OTHER_DOC = "anyOtherDoc";
     public static final String POSITION_STATEMENTS = "positionStatements";
+    public static final String FM5_STATEMENTS = "fm5Statements";
```

Comment:
Hi @JamesCutts00 We don't need this as we are not using it.
Could you please remove this if you are not using it.